config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'creditors/index'
```

Comment:
Let's change these routes to use [resources](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)